sv.c

```diff
@@ -17878,6 +17876,47 @@ Perl_sv_regex_global_pos_clear(pTHX_ SV *sv)
         mg->mg_len = -1;
 }
 
+/*
+=for apidoc sv_vstring_get
+
+If the given SV has vstring magic, stores the string pointer and length into
+the variables addressed by C<pvp> and C<lenp>, and returns true.  If not,
+returns false.
```

Comment:
I suggest a sentence saying,
```
stores the string pointer and length into
the variables addressed by C<pvp> and C<lenp>. The lifespan
of string pointer is until the next FREETMPS;.
```
or
```
The lifespan of string pointer is until next LEAVE;.
```
or
```
The lifespan of string pointer is of undefined storage and from
undefined allocator.   Memcpy() it ASAP if necessary.  It is not documented
 if string pointer is backed by a Newx()/Safesysmalloc() block,
somewhere into the middle of a Newx() block, or is a SVPV COW,
or is a shared string table HEK or is a temp ptr owned by MORTAL
or SAVESTACK or is a process global ptr to a C const "" string literal.
```

Version # 3  documents lifetime as TBD/opaque/UB.

Reply:
None of those apply; but I've now added some words to point out the string buffer is owned by the SV itself (or at least, the magic annotation on it)